src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -752,8 +801,8 @@ public Pattern getIssuePattern() {
      * @return null
      *         if no such was found.
      */
-    public static JiraSite get(Job<?, ?> p) {
-        JiraProjectProperty jpp = p.getProperty(JiraProjectProperty.class);
+    public static JiraSite get(Job<?, ?> job) {
```

Comment:
why renaming this? it's confusing as it generate some noise for nothing. (i.e make me feeling what happen here a lot of changes!)